app/models/api/vatapplication/OverseasCompliance.scala

```diff
@@ -18,9 +18,9 @@ package models.api.vatapplication
 
 import play.api.libs.json.{Format, Json}
 
-case class OverseasCompliance(goodsToOverseas: Boolean,
+case class OverseasCompliance(goodsToOverseas: Option[Boolean],
                               goodsToEu: Option[Boolean],
-                              storingGoodsForDispatch: StoringGoodsForDispatch,
+                              storingGoodsForDispatch: Option[StoringGoodsForDispatch],
```

Comment:
these two fields are already in an optional() in the respective block builder, but now that they're optional themselves the code in those needs to be updated to change .map to .flatMap to avoid Option[Option[]] issues.